content/docs/appflow/package/cli.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: fluid/pro_docs_base
+category: pro-package
+id: pro-native-builds
+title: iOS Credentials - Ionic Appflow Documentation
+body_class: 'pro-docs'
+hide_header_search: true
+dark_header: true
+---
+
+# Package builds on Appflow using the Ionic CLI
+
+It is possible to trigger package builds on Appflow directly from the CLI. 
+This is extremely useful if you need to integrate the package build step inside an existing external CI/CD process.
+
+### Prerequisites
+
+To trigger package builds on Appflow from the Ionic CLI, a proper subscription to run package builds is needed. 
+
+To use environments, a proper subscription that allows the usage of Automations is needed.
+
+### Authentication
+
+The Ionic CLI has to be authenticated on Appflow to be able to trigger a package build.
+
+To login type
+```bash
+$ ionic login <email> <password>
+```
+
+this will prompt for the Appflow username/password.
+
+The CLI can be authenticated also via environment variable (for example if it is used during a CI/CD task).
+
+To set up the authentication via environment variable, first login with an user who has access to the app you want to trigger 
+package builds for for
+
+```bash
+$ ionic login <email> <password>
+```
+
+then export the authentication token 
+
+```bash
+$ ionic config get -g tokens.user
+```
+
+finally export the token in an environment variable of your shell or your CI/CD service.
+
+```bash
+$ export IONIC_TOKEN=<your token>
+```
+
+### Link the app
+
+Your app must be linked to the remote app on Appflow. If the app is not linked yet, just run
+
+```bash
+$ ionic link <app id>
+```
+
+and commit the changes to `ionic.config.json`.
+
+
+### Trigger a package build
+
+Currently package build can be triggered only from the app project directory.
+
+```bash
+$ cd /path/to/your/app
+```
+
+Assuming the CLI is authenticated, to trigger a package build run
+
+```bash
+$ cd ionic package build
```

Comment:
I think the `cd` part is a copy pasta typo

Reply:
right